monitoring/dashboards/README.md

```diff
@@ -0,0 +1,30 @@
+# Building new dashboards
+
+## From OSS
+
+1. Download the OSS JSON definition
+2. Import it in a deployed Grafana
+3. Tweak to your liking
+
+## From scratch
+
+1. Create a new dashboard in a deployed Grafana
+2. Create and tweak panels
+
+
+## Final steps
+
+1. Copy the JSON model
+2. Apply the following modifications:
+   * add the `__inputs` of the original download to the model
+   * add the `__requires` of the original download to the model
+   * move `id` from int to `null` in the model
```

Comment:
I would add: `* remove "uid" entry`

Reply:
Is this OK? I remember we weren't sure about removing or leaving as is yesterday on Slack.

Reply:
I set it to `null`, like id...

Reply:
I don't see any mentions of *UID* but only *ID* did you mixed them up perhaps ?

Reply:
nop, there are 2 fields. But maybe it depends on where the dashboard comes from, or the version...
(See lines 109 and 9028 in https://github.com/scality/Zenko/pull/1401)